pkg/logicalbackup/logicalbackup.go

```diff
@@ -111,9 +111,9 @@ func New(ctx context.Context, stopCh chan struct{}, cfg *config.Config) (*Logica
 		dbCfg:                  pgxConn,
 		replMessageWaitTimeout: waitTimeout,
 		statusTimeout:          statusTimeout,
-		relations:              make(map[message.NamespacedName]message.Relation),
-		relationNames:          make(map[dbutils.Oid]message.NamespacedName),
-		types:                  make(map[dbutils.Oid]message.Type),
+		registeredRelations:    make(map[message.NamespacedName]message.Relation),
+		relationOIDToNames:     make(map[dbutils.Oid]message.NamespacedName),
```

Comment:
what was wrong with the old name?
it had `Oid` as a key and `NamespacedName` as value, pretty clear.